### PR TITLE
Add motor parameters

### PR DESCRIPTION
**Pull request by TheCoolKidsTable**
This PR adds:

-  Parameters for the MX106 and MX64 servos and assigns them to the correct motors. (MX106 for hip,knee,ankles and MX64 for arms/neck)
- PID gains for the motors (Obtained from Simscape)
- Sets selfCollision field for robot to TRUE

TODO:

- [x]  Add backlash

### CODE REVIEW

File: protos/NUgus.proto [hunk 1]
```diff
@@ -1243,4 +1314,4 @@ PROTO NUgus [
       ]
     }
   }
-}
+}
```
Comment (ysims) [inline protos/NUgus.proto:1317]: newline? :smile: Otherwise looks good

**Review by ysims — COMMENTED** [review]
Might need someone who can review the actual values :smile:

**Comment by xXBenjaminYoungXx** [comment]
Add comments on which Dynamixel model is being represented on the Rotational Motor Object.

**Review by Bidski — CHANGES_REQUESTED** [review]
I think you should move the `boundingObject` changes to a separate PR. 

As for the backlash, I think we might find that they are expecting this as a measure of radians (because it is a HingeJoint). In which case 0.33 degrees (from the docs) is 0.005759587 radians (0.006). Have we tested with this value? Ludovic does warn against using a value that is too small

**Review by Bidski — APPROVED** [review]
LGTM

**Review by Bidski — DISMISSED** [review]
Seems ok to me

**Review by ysims — APPROVED** [review]
LGTM

**Pull request merged**